workloads/hyper-scale/README.md

```diff
@@ -0,0 +1,31 @@
+# Hypershift perfscale Scripts
+
+The purpose of the script is to install Hypershift Operator on an Openshift cluster and create HostedCluster workload on them.
+
+1. Install and create cluster: `./run.sh build`
```

Comment:
It might be a convenient option to also have this install the base hypershift cluster. Doesn't necessarily need to be in this PR but as a thought for the future.

Reply:
you mean a script to install a base OSD(AWS) management cluster ? I am not sure if this repo is the right place to keep cluster install scripts, I can write up one and keep it in a separate repository. 
I treat this script as a hypershift workload script to spin/clean up `HostedControlPlane` and `HostedCluster` resources on an existing management cluster.